syntax = "proto3";

package hello;

service HelloService {
  rpc SayHello (HelloRequest) returns (HelloResponse);
}

message HelloRequest {
  string name = 1;
  int32 age = 2;
}

message HelloResponse {
  string yourName = 1;
  string message = 2;
}